### PR TITLE
Updated jQuery version for Chat and Form widgets

### PR DESCRIPTION
**Pull request by mantas**
jQuery version update for 4.2 (next-next at the time of writing) release.

### CODE REVIEW

**Comment by MrGeneration** [comment]
Oh boy sorry. 
You found my little trap - could you please checkout `master` on your branch and then provide the PR against `master`?
We're currently not using `develop` on the documentations - the only reason why we have it (right now) is the protected branch functionality.

**Comment by mantas** [comment]
Moved to `master`!

**Review by MrGeneration — APPROVED** [review]
Thanks!
Will merge during release.

**Comment by MrGeneration** [comment]
**Does not come with 4.1**

File: channels/chat.rst [hunk 1]
```diff
@@ -68,7 +68,7 @@ The final result will look like the following:
 
 Zammad chat requires jQuery. If you don't already use it on your website include it like this:
 
-<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```
Comment (techerella) [inline channels/chat.rst:71]: "IP addresses" (with a double d) --> line 80

Comment (techerella) [inline channels/chat.rst:71]: "You can also find.... in the admin area" (location at the end of the sentence)  --> line 84

Comment (MrGeneration) [reply channels/chat.rst:71]: Hang in there re-arranging the lines while I'm at it for better readability code wise.

Comment (MrGeneration) [reply channels/chat.rst:71]: Hang in there re-arranging the lines while I'm at it for better readability code wise.

**Pull request merged**